goodreads-predictor/src/goodreads_predictor/pipelines/feature_engineering/pipeline.py

```diff
@@ -5,9 +5,13 @@
 
 from kedro.pipeline import Pipeline, pipeline, node
 from .nodes import apply_book_attributes
+from .nodes import merge_description_embeddings
+from .nodes import perform_clustering_analysis
 
 
 def create_pipeline(**kwargs) -> Pipeline:
     return pipeline([
-        node(func=apply_book_attributes, inputs=['filtered_books'], outputs=['books_features', 'feature_cutoffs'], name='add_engineered_features')
+        node(func=apply_book_attributes, inputs=['filtered_books', 'description_embeddings'], outputs=['books_features', 'feature_cutoffs'], name='add_engineered_features'),
```

Comment:
Adding description_embeddings as an input to `apply_book_attributes` doesn't currently work since it is not part of the function signature in the nodes.py script. Please remove this as a function input if it is not being used in this function